feature/dashboard/src/main/res/values/attrs.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <declare-styleable name="FragmentContainerView">
```

Comment:
Maybe name this `SyncFragmentContainerView` to avoid clash with any other fragment container we might choose to parametrize in the future.